test/regression.uts

```diff
@@ -6483,6 +6501,8 @@ test_openbsd_6_3()
 + Mocked _parse_tcpreplay_result(stdout, stderr, argv, results_dict)
 ~ mock_parse_tcpreplay_result
 
+= Test mocked _parse_tcpreplay_result
```

Comment:
Random fix: this test is skipped as missing a `=` flag (see in the logs: WARNING unknown [...]`)